server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/SearchAction.java

```diff
@@ -110,7 +120,55 @@ public void handle(Request request, Response response) {
       }
 
       writeProtobuf(createResponse.build(), request, response);
+    }
+  }
+
+  private SearchAction.Association find(DbSession dbSession, OrganizationDto organization, ProjectQgateAssociationQuery query) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "organization". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)